packages/material-ui-codemod/src/v5.0.0/variant-prop.test/expected.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Select from '@material-ui/core/Select';
+import FormControl from '@material-ui/core/FormControl';
+
+export default function TextFieldComponent(props) {
+  return (
+    <div>
+      <TextField {...props} variant="standard" />
```

Comment:
```suggestion
      <TextField variant="standard" {...props} />
```

Shouldn't this be the result? Otherwise, we are always overriding any `variant` being part of `props`. The same would be applied to the other components if this is correct.

Reply:
Your comment seems correct to me - the codemod was originally introduced by @mbrookes and I basically multiplied it.

Reply:
Ah I see, ok let's consider fixing it in a separate PR.